VMAccess/manifest.xml

```diff
@@ -2,7 +2,7 @@
 <ExtensionImage xmlns="http://schemas.microsoft.com/windowsazure">
   <ProviderNameSpace>Microsoft.OSTCExtensions</ProviderNameSpace>
   <Type>VMAccessForLinux</Type>
-  <Version>1.5.4</Version>
+  <Version>1.5.5</Version>
```

Comment:
This is a big change to release as a patch.  I don't know the current breakdown for extensions with autoUpgradeMinorVersion, but you can release as 1.6 now and verify it works then re-release as 1.5.5 if necessary.

Reply:
That is a great idea. I'll keep that in mind.